lib/shoulda/matchers.rb

```diff
@@ -1,3 +1,5 @@
+require "active_support/core_ext/object/try"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.